onnx/defs/schema.cc

```diff
@@ -5,6 +5,8 @@
 #include <stdexcept>
 #include <unordered_set>
 #include "onnx/checker.h"
+#include "onnx/defs/operator_sets.h"
+#include "onnx/defs/operator_sets-ml.h"
```

Comment:
Need to guard this include with ONNX_ML for non ONNX_ML builds

Reply:
The headers had internal conditionals, but I can add them here.